extensions/dacpac/src/wizard/pages/extractConfigPage.ts

```diff
@@ -111,8 +111,9 @@ export class ExtractConfigPage extends DacFxConfigPage {
 			required: true
 		}).component();
 
-		// default filepath
+		// default version
 		this.versionTextBox.value = '1.0.0.0';
+		this.versionTextBox.ariaLabel = localize('dacfx.versionEdit', "Version Edit");
```

Comment:
do you need to say **edit** - I think NVDA adds edit. Just check one that there is not "edit edit"

Reply:
yeah I wasn't sure about that too. The expected text in the issue says edit. Should we ask for clarification on this?

![image](https://user-images.githubusercontent.com/31145923/63204084-438d7380-c049-11e9-858c-8dddea4b5f65.png)